app/services/gather_bot_stats.rb

```diff
@@ -9,9 +9,9 @@ def self.run(bot, period = 'today')
 
     skills = bot.skills.index_by(&:id)
     {
-      active_users: summary['active_users'],
-      messages_sent: summary['messages_sent'],
-      messages_received: summary['messages_received'],
+      active_users: summary['active_users'] || 0,
```

Comment:
NB: This is fixed in the engine already. I am not sure we need to be defensive here. up to you.

Reply:
It doesn't hurt IMO.